src/amo/components/AddonMoreInfo/index.js

```diff
@@ -203,6 +203,24 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
           </Link>
         </li>
       ),
+      tagsLinks:
+        addon.tags != null && addon.tags.length > 0
+          ? addon.tags.map((tag_text) => {
```

Comment:
```suggestion
          ? addon.tags.map((tagText) => {
```

Reply:
changed `tag_text` to `tagText` everywhere https://github.com/mozilla/addons-frontend/pull/10798/commits/44bb2532ba6466db46ed2818c0df3985fee0b8b0